src/NHibernate.Test/NHSpecificTest/GH1594/ExecutionContextExtensions.cs

```diff
@@ -7,7 +7,12 @@ public static class ExecutionContextExtensions
 	{
 		public static int LocalValuesCount(this ExecutionContext c)
 		{
-			var f = typeof(ExecutionContext).GetField("_localValues", System.Reflection.BindingFlags.NonPublic | System.Reflection.BindingFlags.Instance);
+#if NETSTANDARD1_3 || NETSTANDARD1_6 || NETSATNDARD2_0 || NETCOREAPP2_0
+			const string localValuesFieldName = "m_localValues";
```

Comment:
We support only 2.0. And we have defined a NETFX variable you can use instead, of course with opposite semantic.

Reply:
Changed to use NETFX.